dataprofiler/profilers/profile_builder.py

```diff
@@ -780,6 +780,76 @@ def update_profile(self, data, sample_size=None, min_true_samples=None):
 
         self._update_profile_from_chunk(data, sample_size, min_true_samples)
 
+    def _remove_data_labelers(self):
+        """
+        Helper method for removing all data labelers before saving to disk.
+
+        :return: data_labeler use for unstructured labeling
+        :rtype: DataLabeler
+        """
+        data_labeler = None
+
+        # Delete data labeler for profiler
+        # TODO: fix to correct options when unstructured options avail
+        use_data_labeler = True
+        if self.options and isinstance(self.options, ProfilerOptions):
+            data_labeler_options = self.options.structured_options.data_labeler
+            use_data_labeler = data_labeler_options.is_enabled
+        if use_data_labeler \
```

Comment:
I would maybe add another break here and a comment for this code block so that this function is 3 concise if statement blocks that are visually separated, not huge issue, will not block this PR.

Reply:
done